### DISCUSSION TITLE
Running dotnet pack from within a Github Action does not appear to generate version.

### DISCUSSION THREAD

**Discussion by ghorsey** [adamralph/minver#528] [Q&A] [answered]
I'm trying to get github actions working for my open graph library.

When I run the command `dotnet pack -c Release --no-restore --include-symbols` in the action the output is:

    Successfully created package 'OpenGraph-Net.0.0.0-alpha.0.nupkg'.
    Successfully created package 'OpenGraph-Net.0.0.0-alpha.0.snupkg'.

Whereas, running the same command locally the output is what I'd expect: 

    Successfully created package 'OpenGraph-Net.3.2.7-alpha.0.21.nupkg'.
    Successfully created package 'OpenGraph-Net.3.2.7-alpha.0.21.snupkg'.

Is there maybe something I am missing in my very rudimentary [yaml file](https://github.com/ghorsey/OpenGraph-Net/blob/main/.github/workflows/main.yml) to get MinVer to work in that environment?

**Comment by adamralph** [comment, marked as answer]
hi @ghorsey, see https://github.com/adamralph/minver#why-is-the-default-version-sometimes-used-in-github-actions-and-travis-ci-when-a-version-tag-exists-in-the-history
